entities.py

```diff
@@ -0,0 +1,221 @@
+"""
+LibrePCB S-expression entities
+"""
+
+from common import format_float
+from typing import List
+
+
+class DateValue(object):
```

Comment:
We can skip Python 2 support, we're using type annotations which aren't compatible anyways.

```python
class DateValue:
```

Same for all other top level classes.

Reply:
Fixed.